lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -283,15 +283,19 @@ protected function getPortableNamespaceDefinition(array $namespace)
      */
     protected function _getPortableSequenceDefinition($sequence)
     {
-        if ($sequence['schemaname'] != 'public') {
+        if ($sequence['schemaname'] !== 'public') {
             $sequenceName = $sequence['schemaname'] . "." . $sequence['relname'];
         } else {
             $sequenceName = $sequence['relname'];
         }
 
-        $data = $this->_conn->fetchAll('SELECT min_value, increment_by FROM ' . $this->_platform->quoteIdentifier($sequenceName));
+        if(!isset($sequence['increment_by'], $sequence['min_value'])){
+            $data = $this->_conn->fetchAll('SELECT min_value, increment_by FROM ' . $this->_platform->quoteIdentifier($sequenceName));
+            $sequence['increment_by'] = $data[0]['min_value'];
+            $sequence['min_value'] = $data[0]['increment_by'];
```

Comment:
@simPod is this the reason of the test failures?

Reply:
@morozov you're right, thanks! I failed to run those localy but it passes now